src/ramses_tx/parsers.py

```diff
@@ -1950,12 +1950,25 @@ def parser_2e04(payload: str, msg: Message) -> PayDictT._2E04:
     return result  # TODO: double-check the final "00"
 
 
-# presence_detect, HVAC sensor
+# presence_detect, HVAC sensor, or Timed boost for Vasco D60
 def parser_2e10(payload: str, msg: Message) -> dict[str, Any]:
-    assert payload in ("0001", "000100"), _INFORM_DEV_MSG
-
+    presence: str | int
+    if (
+        payload == "00021E04060000"
+    ):  # Vasco remote timed boost on, button bottom right pressed
+        return {
+            "mode": "boost",
+            "duration": (payload[5:6]),
+            "_unknown_4": payload[7:],
+        }
+    elif msg.src.type == "37":  # ClimaRad VenturaV1x FAN
+        assert payload in ("000000", "000100"), _INFORM_DEV_MSG
+        presence = int(payload[2:4])
+    else:
+        assert payload in ("0001", "000100"), _INFORM_DEV_MSG
```

Comment:
000100 seems like a strange value to accept based on the packets in the log that are not from my ClimeRad Ventura